### PR TITLE
Site refresh

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
Manual site refresh for 1.2.0 RC0
Preview: http://107.21.90.138/

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -1,5 +0,0 @@
-RewriteEngine on
```
Comment (marcoabreu) [inline .htaccess]: Is it intended to remove this?

**Comment by eric-haibin-lin** [comment]
Wow.. big change.. How are ppl supposed to review this..?

File: .htaccess (hunk 1 above)
Comment (aaronmarkham) [reply .htaccess]: No. That's not right. I'll see what's up.

**Comment by aaronmarkham** [comment]
Yes, it's too big to review. I'm going to try to split this up by version branch.

**Pull request closed without merging**